src/modules/commander/accelerometer_calibration.cpp

```diff
@@ -177,7 +178,6 @@ static calibrate_return read_accelerometer_avg(float (&accel_avg)[MAX_ACCEL_SENS
 		{ORB_ID(sensor_accel), 0, 0},
 		{ORB_ID(sensor_accel), 0, 1},
 		{ORB_ID(sensor_accel), 0, 2},
-		{ORB_ID(sensor_accel), 0, 3},
```

Comment:
I still think up to 4 accels is good (eg typical board with 3 internal + external connected).

Reply:
@dagar , would you like me to bring back the 4 accel/gyro instances and drop the mag/baro to 2 instance?

Reply:
@dagar, I went ahead and put the accel and gyro counts back to 4, mag and baro down to 2.  Commit pushed up.